### ISSUE TITLE
roadmap 

### ISSUE THREAD

**Issue by avgur770** [TeleportXYZ/TRIP-Guides#34]
### What's missing from the TRIP Guides

a roadmap

### Expected content

there should be a roadmap on the guide.
it should also show what roles will be need by genesis city. and explain a little on gradual decentralization 
### Actual content

doesn't have a roadmap

**Comment by Alejandro1954-ops** [comment]
Agreed and it does not have to be anything complicated. Example attached:

![Nibiru Chain](https://github.com/TeleportXYZ/TRIP-Guides/assets/29745428/db4eb3d6-fb97-47de-bc59-7e0799665ef9)

Add quarterly estimate release dates (Q2 2024) and add it as a section to the [guide](https://guides.trip.dev/). 

Added plus that the roadmap can be shared on socials and marketing materials.
